package.json

```diff
@@ -18,7 +18,7 @@
     ]
   },
   "scripts": {
-    "build": "yarn build:internal-dependencies && yarn build:non-internal-dependencies",
+    "build": "yarn workspaces foreach -ptv --jobs unlimited --exclude tupaia run build",
```

Comment:
p: parallel
t: topological (leaf nodes first, then parents etc)
v: verbose, prefixes each line with the package name
--jobs unlimited: use all the cpus you have
--exclude tupaia: avoid re-running build in root package